OpenSSL/test/test_ssl.py

```diff
@@ -2129,6 +2129,20 @@ def test_get_cipher_bits(self):
         self.assertEqual(server_cipher_bits, client_cipher_bits)
 
 
+    def test_get_protocol_version(self):
+        """
+        :py:obj:`Connection.get_protocol_version` returns a :py:class:`int`
+        giving the protocol version of the current connection.
+        """
+        server, client = self._loopback()
+        server_protocol_version, client_protocol_version = \
+            server.get_protocol_version(), client.get_protocol_version()
+
+        self.assertIsInstance(server_protocol_version, int)
+        self.assertIsInstance(client_protocol_version, int)
+
+        self.assertEqual(server_protocol_version, client_protocol_version)
```

Comment:
This assertion would be satisfied if the implementation accidentally always returned a constant integer (`0` for example).

Perhaps this test could make a stronger assertion about the value?  It might even be worth having a couple tests for a couple different values.